source/simulator/parameters.cc

```diff
@@ -657,6 +658,40 @@ namespace aspect
                            "is largely empirically decided -- it must be large enough to ensure "
                            "the bilinear form is coercive, but not so large as to penalize "
                            "discontinuity at all costs.");
+        prm.declare_entry ("Use limiter for discontinuous temperature solution", "false",
+                           Patterns::Bool (),
+                           "Whether to apply the bound preserving limiter as a correction after having"
+                           "the discontinous temperature solution. Currently we apply this only to the"
+                           "temperature solution if the global max/min are already determined and defined"
+                           "This limiter keeps the discontinuous solution in the range given by"
+                           "Global temperature max or min.");
```

Comment:
Put single quotes around `Global temperature max` to indicate that the three words are a unit. Maybe replace `min` by `'Global temperature min'`.